### ISSUE TITLE
Miner closes off after executed crypted .exe

### ISSUE THREAD

**Issue by Untorest** [UnamSanctam/SilentCryptoMiner#2373]
Unam, pls i need help. The miner worked very well.
But my crypter seller crypted it for test and the crypted miner.exe just closes off and not start mining.
This is what the crypter seller assumed it as problem:
"The miner asks for privileges at later stages of runtime, my crypt gets the first ones. 
Maybe it fails for the last two prompts somewhere"
What if I disable the "Run as administrator" function and crypt again, would the problem be solved?

Or is that because I enabled Startup function in the miner builder instead of crypter's startup function? I tested it as builder's startup function ON, Crypter's startup OFF

**Comment by UnamSanctam** [comment]
If it's an injection crypter (like most are) and if it does not inject into itself then you need to use the crypter Startup function instead of the miners, because when the miner is injected then it has no idea where the original file is and thus can't copy it to the installation folder.

**Comment by Untorest** [comment]
![Screenshot_2024-05-04_175020](https://github.com/UnamSanctam/SilentCryptoMiner/assets/157064340/9101a619-c0ac-4be0-b0e7-b2a4fc4202c5)
![Screenshot_2024-05-04_175001](https://github.com/UnamSanctam/SilentCryptoMiner/assets/157064340/23301ce8-37b9-49c4-89c3-33b9eebb868b)

**Comment by UnamSanctam** [comment]
Yes that should likely work, though for your real build a "Remote Configuration" (or several for backups) is of course recommended in case you ever need to change the settings.

**Comment by Untorest** [comment]
I disabled startup, but it still doesn't working after crypted. Would that because the crypter and miner.exe BOTH support injecting  in some files like conhost.exe, explorer.exe?

**Comment by UnamSanctam** [comment]
Sorry for the late response, I've been sick. The injection shouldn't be able to cause any problems, the only theoretical problem is the "Startup". Does the crypt work if you don't have any options enabled in the miner?

**Comment by s1ck-1337** [comment]
I got exactly the same problem....
Miner is running well but after crypting it doesnt execute.
I already tried to disable the startup within the miner and let the crypter do it, but it doesnt work.
I asked the developer of the crypter and he said the crypter is not compatible with the miner because it has own injection method.

Anything we can do about it ?

**Comment by UnamSanctam** [comment]
It does not work even with these settings?

[TESTSETTINGS.zip](https://github.com/UnamSanctam/SilentCryptoMiner/files/15326090/TESTSETTINGS.zip)

**Comment by s1ck-1337** [comment]
I tried those settings. Nope :(
I use Data-Encoder for encryption but cant get it to work.
Encryption is working but it wont execute the miner

**Comment by tarekxxx** [comment]
> I tried those settings. Nope :( I use Data-Encoder for encryption but cant get it to work. Encryption is working but it wont execute the miner



> I tried those settings. Nope :( I use Data-Encoder for encryption but cant get it to work. Encryption is working but it wont execute the miner

DM  TG @XHaDoTTaaX

**Comment by s1ck-1337** [comment]
No, I havent tried yet.
Will do now

**Comment by s1ck-1337** [comment]
Doesn't work. I really tried every combination in the settings of the miner AND the crypter. 
The Data-Encoder Crypter is a very good one but sadly not working with this miner... 60 Bucks trashed :(
Can someone recommend a not too expensive crypter that will work with this miner ?
